### PR TITLE
Release v1.5.0

### PR DESCRIPTION
**Pull request by trezy**
## ✨New features

* [[WEB-72]](https://rollforguild.atlassian.net/browse/WEB-72) Dialogs now have their own component!
* [[WEB-84]](https://rollforguild.atlassian.net/browse/WEB-84) We have tests!
* Group search is now available even if the user isn't logged in

## 🐛Bug fixes

* [[WEB-89]](https://rollforguild.atlassian.net/browse/WEB-89) Alerts no longer show up under headers
* [[WEB-90]](https://rollforguild.atlassian.net/browse/WEB-90) Groups with no description now have a placeholder description in user profiles
* [[WEB-93]](https://rollforguild.atlassian.net/browse/WEB-93) The forum now has a placeholder if there are no threads
* [[WEB-94]](https://rollforguild.atlassian.net/browse/WEB-94) Threads now have a placeholder if there are no comments
* [[WEB-95]](https://rollforguild.atlassian.net/browse/WEB-95) Usernames can no longer be created with spaces (sneaky users...)

### CODE REVIEW

File: nodemon.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "ext": "js,json",
```
Comment (UncleClapton) [inline nodemon.json]: If removing nodemon, we should remove this

File: tagmanager.config.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "development": {
```
Comment (UncleClapton) [inline tagmanager.config.json]: should we have this in repo?

Comment (trezy) [reply tagmanager.config.json]: Yup, it's not sensitive information. It has no value other than telling the config which Tag Manager environment to use.

File: nodemon.json (hunk 1 above)
Comment (trezy) [reply nodemon.json]: For sure, I'll pull it out in the branch where I'm trying to get tests to run in CI.

**Pull request merged**